### ISSUE TITLE
ReferenceError: t2 is not defined

### ISSUE THREAD

**Issue by vasyl-shumskyi** [krocon/find-duplicate-files#3]
Hello, 

Looks like this module seems to be unusable because of following tiny error:
```javascript
  var dt = (Date.now() - t2);
                                               ^

ReferenceError: t2 is not defined
    at createMissingMd5 (/opt/bin/tiny-duplicate-finder/node_modules/find-duplicate-files/index.js:121:48)
    at start (/opt/bin/tiny-duplicate-finder/node_modules/find-duplicate-files/index.js:193:17)
    at /opt/bin/tiny-duplicate-finder/node_modules/find-duplicate-files/index.js:209:13
    at module.exports (/opt/bin/tiny-duplicate-finder/node_modules/find-duplicate-files/index.js:211:11)
    at Object.<anonymous> (/opt/bin/tiny-duplicate-finder/test/dd.js:6:1)
    at Module._compile (module.js:569:30)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:503:32)
    at tryModuleLoad (module.js:466:12)
    at Function.Module._load (module.js:458:3)
```

Following default code snippet was used to get above error:

```javascript
var dir = '/Users/Username/Desktop'

var findDuplicateFiles = require('find-duplicate-files');

// Basic usage (a default callback handler will log the results):
findDuplicateFiles(dir, {});
```

**Comment by krocon** [comment]
fixed.

happy xmas!
Marc

**Comment by vasyl-shumskyi** [comment]
Wow, that was fast :) 

Thanks and Happy Xmas!

**Comment by danepowell** [comment]
Looks like this fix was never pushed to npm (v0.0.6)

**Comment by krocon** [comment]
you're right. I just updated (pubslished) the new version 0.0.7.  thx!
